src/responses.ts

```diff
@@ -97,3 +97,31 @@ export interface GetTransactionStatusRpcResponse {
   }
   receipts_outcome: any[]
 }
+
+export interface GetBlockWithChunkRpcResponse {
+  result: {
+    header: {
+      chunk_hash: string
+      prev_block_hash: string
+      outcome_root: string
+      prev_state_root: string
+      encoded_merkle_root: string
+      encoded_length: number
+      height_created: number
+      height_included: number
+      shard_id: number
+      gas_used: number
+      gas_limit: number
+      rent_paid: string
+      validator_reward: string
+      balance_burnt: string
+      outgoing_receipts_root: string
+      tx_root: string
+      validator_proposals: any[]
+      signature: string
+    }
+    transactions: []
+    receipts: []
```

Comment:
let's do `any[]` if we are not going to put the whole interface there (although since we are returning the whole thing we probably should)

Reply:
Okay!